src/main/java/com/docker/jersey/restapi/DockerRegistrationApi.java

```diff
@@ -0,0 +1,31 @@
+package com.docker.jersey.restapi;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+import java.util.Optional;
+
+@Path("/dockerRegistration")
+public class DockerRegistrationApi {
+    @POST
+    public Response createImage() {
+        return Response.status(501).entity(Optional.empty()).build();
+    }
+
+    @GET
+    @Path("/{id}")
```

Comment:
here is a mismatch of param name with other endpoints
also "dockerId" might be better naming
also type String might be better in all cases, due to its conversion capabilities and immutability